LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -210,28 +210,12 @@ public LoRaDevice(string devAddr, string devEUI, ILoRaDeviceClientConnectionMana
         /// </summary>
         public async Task<bool> InitializeAsync()
         {
-            Twin twin;
-            try
-            {
-                twin = await this.connectionManager.GetClient(this)?.GetTwinAsync();
-            }
-            catch (IotHubCommunicationException ex)
-            {
-                Logger.Log(this.DevEUI, $"Error while communication with IoT Hub during device initialization. {ex.Message}", LogLevel.Error);
-                throw;
-            }
-            catch (IotHubException ex)
-            {
-                Logger.Log(this.DevEUI, $"An error occured in IoT Hub during device initialization. {ex.Message}", LogLevel.Error);
-                throw;
-            }
+            var twin = await this.connectionManager.GetClient(this)?.GetTwinAsync();
```

Comment:
I guess the interception of this error is now in the main loop right?

Reply:
Exactly. This error will only be intercepted on the top level, in order to not log exceptions multiple times.